actions/java-gradle-assess-code-quality/action.yaml

```diff
@@ -70,8 +79,8 @@ runs:
 
     # sonar.branch.autoconfig.disabled is necessary for autocreation of projects on Sonarcloud to work
     - name: Assess code quality
-      if: ${{ inputs.sonar-token != '' && inputs.sonar-organization != '' }}
-      run: ./gradlew -Dsonar.branch.autoconfig.disabled=true -Dsonar.host.url=https://sonarcloud.io -Dsonar.organization=${{ inputs.sonar-organization }} --info --stacktrace sonarqube ${{ inputs.gradle-refresh-dependencies == 'true' && '--refresh-dependencies' || '' }}
+      if: ${{ inputs.sonar-token != '' }}
+      run: ./gradlew -Dsonar.branch.autoconfig.disabled=true -Dsonar.host.url=${{ inputs.sonar-host }} ${{ (inputs.sonar-organization != '' && format('-Dsonar.organization={0}', inputs.sonar-organization)) || '' }} ${{ (inputs.sonar-project-key != '' && format('-Dsonar.projectKey={0}', inputs.sonar-project-key)) || '' }} ${{ (inputs.sonar-project-name != '' && format('-Dsonar.projectName={0}', inputs.sonar-project-name)) || '' }} --info --stacktrace sonarqube ${{ inputs.gradle-refresh-dependencies == 'true' && '--refresh-dependencies' || '' }}
```

Comment:
Can you break this onto multiple lines to increase readability?

Reply:
Done